classes/class-p4-search.php

```diff
@@ -322,11 +322,13 @@ public function get_paged_posts() {
 					// Handle submitted filter options.
 					if ( $selected_filters && is_array( $selected_filters ) ) {
 						foreach ( $selected_filters as $type => $filter_type ) {
-							foreach ( $filter_type as $name => $id ) {
-								$filters[ $type ][] = [
-									'id'   => $id,
-									'name' => $name,
-								];
+							if ( is_array( $filter_type ) ) {
```

Comment:
It's a bit easier to read if you `continue` when it's not an array, so the rest of the loop doesn't need to be wrapped in a long if-statement.

Reply:
yes, I agree with you 👍